spec.md

```diff
@@ -52,46 +52,44 @@ On a theoretical front, Val owes greatly to linear types [(Wadler 1990)](https:/
 1. The Boolean literals are `true` and `false`:
 
     ```ebnf
-    boolean-literal ::=
-      true
-      false
+    boolean-literal ::= (one of)
+      true false
     ```
 
 #### Integer literals
 
 1. Integer literals have the form:
 
     ```ebnf
-    integer-literal ::=
+    integer-literal ::= 
```

Comment:
Seems there's a trailing whitespace?

Reply:
Worrying about trailing WS is a huge waste of everybody's time, but if that bugs you I will avoid introducing it.

Reply:
Never mind. I once worked with someone who rejected my commits as soon as there were trailing spaces, or not exactly one newline at the end of a file, so now I look for this stuff. I personally don't mind much.

Reply:
I just did a mechanical cleanup of trailing WS and there are already many lines consisting of only nonempty trailing whitespace.  FWIW.

Reply:
Well, that's embarrassing. But as I said, I don't mind much about trailing whitespaces.

Reply:
Too late; it's all gone.